src/Tribe/Integrations/Manager.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__Manager
+ *
+ * Loads and manages the third-party plugins integration implementations.
+ */
+class Tribe__Events__Integrations__Manager {
+
+	/**
+	 * @var Tribe__Events__Integrations__Manager
+	 */
+	protected static $instance;
+
+	/**
+	 * The class singleton constructor.
+	 *
+	 * @return Tribe__Events__Integrations__Manager
+	 */
+	public static function instance() {
```

Comment:
If we want to ensure only one instance of this class can be created we should probably define a protected or private constructor (with an empty body). Otherwise, accidents can happen in future where other code instantiates additional objects of this type (we just saw this with Tribe__Main for example).